requirements-test.txt

```diff
@@ -0,0 +1,13 @@
+async_generator
```

Comment:
Let's move all the dependencies to `requirements.txt` file - it's easy to update and use instead of running `setup.py` each time